composer.json

```diff
@@ -14,6 +14,7 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=5.3.9",
+        "doctrine/orm": "~2.2,>=2.2.3,<2.5",
```

Comment:
2.5 has a BC break with 2.4 with the setup used by the Symfony standard (you had to configure some new setting, otherwise the DB connection couldn't be made, breaking all Symfony pages, as connections were always made).

For a package, 2.5 can be completely valid. When changing to `^2.2.3`, please also update the rest of the dependencies to all use the same operators (for consistency, I'm slowly upgrading all CMF packages to use `^` instead of `~`).